docs/blog/pydantic-v2-alpha.md

```diff
@@ -133,7 +133,7 @@ a full migration guide, but for now the following pointers should be some help w
 
 The following config settings have been removed:
 
-* `allow_mutation`.
+* `allow_mutation` — this has been removed. You should be able to use [frozen](https://docs.pydantic.dev/dev-v2/api/config/#pydantic.config.ConfigDict) equivalently (inverse of current use).
```

Comment:
We don't need this change in the blog post.

Please revert this change and change the migration guide in https://github.com/pydantic/pydantic/blob/c2fe0b87e1cbd09a58fb30be5ec6bbb372280c45/docs/migration.md?plain=1#L145

Reply:
Also, it's better to use a relative link
[frozen](api/config.md#pydantic.config.ConfigDict)

Reply:
Sure - would it not be helpful in the blog post as well? Appreciate the migration guide is the more important of the two

Reply:
Ok, let's do it in both places

Reply:
Resolved